lib/ecs_deploy/service.rb

```diff
@@ -172,13 +173,23 @@ def self.wait_all_running(services)
         client = Aws::ECS::Client.new(region: region)
         ss.reject(&:delete).map(&:service_name).each_slice(MAX_DESCRIBE_SERVICES).map do |chunked_service_names|
           Thread.new do
+            last_events = {}
             EcsDeploy.config.ecs_wait_until_services_stable_max_attempts.times do
               EcsDeploy.logger.info "wait service stable [#{chunked_service_names.join(", ")}] [#{cl}]"
               resp = client.describe_services(cluster: cl, services: chunked_service_names)
               resp.services.each do |s|
                 # cf. https://github.com/aws/aws-sdk-ruby/blob/master/gems/aws-sdk-ecs/lib/aws-sdk-ecs/waiters.rb#L91-L96
                 if s.deployments.size == 1 && s.running_count == s.desired_count
                   chunked_service_names.delete(s.service_name)
```

Comment:
It is better to display the remaining events here. Otherwise, they are never displayed.

Reply:
Fixed

```console
❯ bundle exec cap production ecs:deploy
I, [2022-12-19T14:32:53.725901 #2812599]  INFO -- : register task definition [okkez-sandbox-app] [ap-northeast-1] [OK]
I, [2022-12-19T14:32:54.214177 #2812599]  INFO -- : update service [okkez-sandbox-app] [okkez-sandbox-cluster-2] [ap-northeast-1] [OK]
I, [2022-12-19T14:32:54.217652 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:33:09.261750 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:33:24.553184 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:33:39.672860 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:33:54.802185 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:34:09.938256 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:34:25.159964 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:34:40.286897 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:34:55.410169 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:35:10.533126 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:35:25.741757 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:35:40.858548 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:35:41.094953 #2812599]  INFO -- : (service okkez-sandbox-app, taskSet ecs-svc/0449769836334102730) has started 6 tasks: (task 33d41f97de444922a80315426c44dc74) (task 6b916a397a6b4fcbb23752880f087b6f) (task 8aa6ec810d344d0281b1c7f9f044515c) (task e901ed4c491e412cb44ab74e50c5e51c) (task eeda3f1bfdc1475cbfcc09da2652ba6b) (task faa1f570b0614c34a6ab28ccedc82219).
I, [2022-12-19T14:35:56.095204 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:36:11.220925 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:36:26.437306 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:36:41.555398 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:36:41.668756 #2812599]  INFO -- : (service okkez-sandbox-app) has stopped 6 running tasks: (task faa1f570b0614c34a6ab28ccedc82219) (task eeda3f1bfdc1475cbfcc09da2652ba6b) (task 8aa6ec810d344d0281b1c7f9f044515c) (task 33d41f97de444922a80315426c44dc74) (task e901ed4c491e412cb44ab74e50c5e51c) (task 6b916a397a6b4fcbb23752880f087b6f).
I, [2022-12-19T14:36:56.668989 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:37:11.903274 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:37:12.125266 #2812599]  INFO -- : (service okkez-sandbox-app) has started 6 tasks: (task 02109afdd42d45708a175823dbd4a512) (task 1551e23331dd4ddbbdf50584e8ce647a) (task 91109c4a5a2e45a6bfb18c76a016e2c0) (task 60e9be436ca548699dd74bd95f732812) (task bce784990490409eab5920074b85c841) (task 9e3f60dbd8b74bc08581cd83cc99e054).
I, [2022-12-19T14:37:27.125499 #2812599]  INFO -- : wait service stable [okkez-sandbox-app] [okkez-sandbox-cluster-2]
I, [2022-12-19T14:37:27.351945 #2812599]  INFO -- : (service okkez-sandbox-app, taskSet ecs-svc/8816674063730807348) has started 6 tasks: (task 02109afdd42d45708a175823dbd4a512) (task 1551e23331dd4ddbbdf50584e8ce647a) (task 60e9be436ca548699dd74bd95f732812) (task 91109c4a5a2e45a6bfb18c76a016e2c0) (task 9e3f60dbd8b74bc08581cd83cc99e054) (task bce784990490409eab5920074b85c841).
I, [2022-12-19T14:37:27.352041 #2812599]  INFO -- : (service okkez-sandbox-app) (deployment ecs-svc/8816674063730807348) deployment completed.
I, [2022-12-19T14:37:27.352083 #2812599]  INFO -- : (service okkez-sandbox-app) has reached a steady state.
```